scripts/reactor_deployment.py

```diff
@@ -8,7 +8,42 @@
 
 
 # # # # # # # # # # # # Constituent Functions # # # # # # # # # # #
+def capacity_increase(df, base_col, rate, start_year, end_year):
+    """
+    This function takes in an increase rate, and creates a new column in the
+    dataframe populated with the increased capacity.
+
+    Parameters
+    ----------
+    df : pandas DataFrame
```

Comment:
```suggestion
    df : :class:`pandas.DataFrame` 
```
this pattern helps formatting in autogenerated docs.